extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -146,13 +149,24 @@ export class FileConfigPage extends ImportPage {
 			return false;
 		}
 
-		let values = await this.getDatabaseValues();
+		let serverDefaultDatabase = this.model.server.options.database;
```

Comment:
And to make sure - this is going to default to the database the user launched the import dialog from?

Reply:
Yes, I'm initially defaulting to the database found in the connection.